app/views/sail/settings/update.js.erb

```diff
@@ -1,5 +1,7 @@
 var messageDiv = $("<%= "#setting-message-#{@setting.name}" %>");
 var cardDiv = messageDiv.closest(".card");
+cardDiv.css('opacity', '.95');
```

Comment:
I really like this idea of changing the opacity after saving. However, .95 is very subtle. What are your thoughts on lowering it a bit? Something like .85.